src/controller_store.coffee

```diff
@@ -0,0 +1,28 @@
+camelCase = require 'camel-case'
+FileIdentifier = require './file_identifier'
+fs = require 'fs'
+path = require 'path'
+
+
+class ControllerStore
```

Comment:
I would add a description to this class. The purpose of a `ControllerStore` is not immediately obvious to the uninitiated reader.